src/main/java/org/opentripplanner/graph_builder/module/osm/OsmDatabase.java

```diff
@@ -1098,37 +1117,36 @@ private void processRoad(OSMRelation relation) {
    * @see "http://wiki.openstreetmap.org/wiki/Tag:public_transport%3Dstop_area"
    */
   private void processPublicTransportStopArea(OSMRelation relation) {
-    OSMWithTags platformArea = null;
-    Set<OSMNode> platformsNodes = new HashSet<>();
+    Set<OSMWithTags> platformAreas = new HashSet<>();
+    Set<OSMNode> platformNodes = new HashSet<>();
     for (OSMRelationMember member : relation.getMembers()) {
       if (
         "way".equals(member.getType()) &&
         "platform".equals(member.getRole()) &&
         areaWayIds.contains(member.getRef())
       ) {
-        if (platformArea == null) {
-          platformArea = areaWaysById.get(member.getRef());
-        } else {
-          issueStore.add(new TooManyAreasInRelation(relation));
-        }
+        platformAreas.add(areaWaysById.get(member.getRef()));
```

Comment:
I think we should make `platformAreas` a `SetMultimap`: https://guava.dev/releases/19.0/api/docs/com/google/common/collect/SetMultimap.html

That way we can also make sure that we don't return null.